debug/org.eclipse.debug.tests/src/org/eclipse/debug/tests/TestsPlugin.java

```diff
@@ -38,23 +39,26 @@ public class TestsPlugin {
 	public static final String PLUGIN_ID = "org.eclipse.debug.tests"; //$NON-NLS-1$
 
 	/**
-	 * Returns the file corresponding to the specified path from within this bundle
-	 * @return the file corresponding to the specified path from within this bundle, or
-	 * <code>null</code> if not found
+	 * Returns the file corresponding to the specified path from within this
+	 * bundle
+	 *
+	 * @return the file corresponding to the specified path from within this
+	 *         bundle, or <code>null</code> if not found
 	 */
-	public static File getFileInPlugin(IPath path) {
+	public static File getFileInPlugin(String path) {
 		try {
 			Bundle bundle = FrameworkUtil.getBundle(TestsPlugin.class);
-			URL installURL = new URL(bundle.getEntry("/"), path.toString()); //$NON-NLS-1$
-			URL localURL= FileLocator.toFileURL(installURL);//Platform.asLocalURL(installURL);
+			URI installURI = new URI(bundle.getEntry("/") + path); //$NON-NLS-1$
```

Comment:
Wouldn't it be simpler to pass the path and use the returned URL directly?
```suggestion
			URL installURL = bundle.getEntry("/" + path);
```